packages/docs/src/page-configs/ui-elements/affix/page-config.ts

```diff
@@ -19,6 +19,9 @@ const config: ApiDocsBlock[] = [
   block.headline('affix.examples.bottom.title'),
   block.example('Bottom'),
 
+  block.subtitle('all.cssVariables'),
+  block.paragraph('affix.variables'),
+
```

Comment:
```suggestion
  block.cssVariables('/va-alert/_variables.scss')
```

You can probably make a high-lvl block.
vuestic-ui/packages/docs/src/helpers/DocsHelper.ts (look at exampleBlock)

Instead of adding it to all translations, you can simply make a "presset". 

We also have block.link.